lib/rules/require-explicit-slots.js

```diff
@@ -95,6 +96,32 @@ module.exports = {
                 slotsDefined.add(slotName)
               }
             }
+          } else if (
+            param.type === 'TSTypeReference' &&
+            param.typeName.type === 'Identifier'
+          ) {
+            const componentPropsFromTypeDefine =
+              getComponentPropsFromTypeDefine(context, typeArguments?.params[0])
```

Comment:
Hmm... I'm not sure if there's any problem using `getComponentPropsFromTypeDefine` since it's just a helper for Props. Can you please fix the issue without using `getComponentPropsFromTypeDefine`?